File: 0014-longest-common-prefix/0014-longest-common-prefix.cpp
class Solution {
public:
    string longestCommonPrefix(vector<string>& strs) {
        sort(strs.begin(),strs.end());
        int n=strs.size();
        string first=strs[0];
        string last=strs[n-1];
        n=min(first.size(),last.size());
        int i=0;
        for(;i<n;i++){
            if(first[i]!=last[i]) break;
        }
        return first.substr(0,i);
    }
};

File: 0014-longest-common-prefix/0014-longest-common-prefix.cs
public class Solution {
    public string LongestCommonPrefix(string[] strs) {
        string prefix=strs[0];
        for(int i=1;i<strs.Length;i++){
            int j=0;
            int n=strs[i].Length;
            while(j<prefix.Length && j<n && strs[i][j]==prefix[j]) j++;
            prefix=prefix.Substring(0,j);
        }
        return prefix;
    }
}